Gathering of coverage data fails with:

Can't combine line data with arc data

--- aiohttp-3.8.6/setup.cfg.orig
+++ aiohttp-3.8.6/setup.cfg
@@ -116,9 +116,6 @@
 	
 	--showlocals
 	
-	--cov=aiohttp
-	--cov=tests/
-	
 	-m "not dev_mode"
 filterwarnings = 
 	error
